pages/desktop/home.py

```diff
@@ -95,6 +88,17 @@ def go_to_page(self):
         },
     ]
 
+    def __init__(self, testsetup):
+        super(HomePage, self).__init__(testsetup)
+        if 'www-dev.allizom.org' not in self.base_url:
```

Comment:
Short of a large refactor I think this is probably the most sensible approach. Could we add a comment to explain why we're doing this though? Also, it's not clear to me if this is temporary or something that will roll out across the environments - could we make that clear in the comment.